ParisBNF/abb/BNFabb149.xml

```diff
@@ -0,0 +1,482 @@
+                           <locus target="#109"/>
+                           <seg type="script">Late 18th- or early 19th-century script.</seg>
+                              <seg type="ink">Black, red.</seg>
+                            <desc>Irregular, by a mediocre scribe.
+                              Right sloping, slender script.</desc>
+                           </handNote>
+                      </handDesc>
+
+                      <additions>
+                        <list>
+                          <item xml:id="a5">
+                            <locus target="#86v"/>
+                             <desc type="Unclear"/>
+                               <q xml:lang="gez">ዝመጽሐፍ፡ ፹፶፩፴፷፴፪፻፪᎓ ወልዱ፡ ፪<gap reason="illegible"/>፶፮፬፮፻፬᎓ ዘወሀቦ፡ ፳፻፪᎓ ፮፯፻፵፪</q>
+                              <note>The note is for the most part composed of numbers possibly
+                              representing encrypted words.</note>
+                                 </item>
+                        <item xml:id="a6">
+                          <locus from="109r" to="109v"/>
+                           <desc type="Unclear"/>
+                             <q xml:lang="gez">ንሰግድ፡ ሰላም፡ ለኪ፡ ሰአሊ፡ ለነ፡ ወዕቀባ፡ ወአሮን፡ ካህን፡ በማዕከላ፡ ያዓርግ፡ ወተስፋ፡</q>
+                            <note>Crudely written.</note>
+                               </item>
+
+                               <item xml:id="e1">
+                                 <locus target="#89r"/>
+                                   <desc>Pen trial.</desc>
+                                   <q xml:lang="gez">ብርዕ፡ ዘተፈትነ፡</q>
+                                   </item>
+
+                                   <item xml:id="e2">
+                                     <locus target="#99r"/>
+                                       <desc>Omitted passage added in the upper margin,
+                                         its place in the text indicated with
+                                         <foreign xml:lang="am">tamallas-sign</foreign> (cross).</desc>
+                                       <q xml:lang="gez">ብርዕ፡ ዘተፈትነ፡</q>
+                                       </item>
+                         </list>
+                       </additions>
+                     </physDesc>
+                     </msPart>
+                   </msPart>
+
+                   <msPart xml:id="p3">
+                      <msIdentifier/>
+                      <msContents>
+                         <summary/>
+                       <msItem xml:id="p3_i1.1">
+                       <locus target="#105"/>
+                       <locus from="108r" to="109v"/>
+                       <locus target="#110"/>
+                       <title type="incomplete" ref="LIT2509Weddas"/>
+                       <incipit xml:lang="gez">ር፡ ሶቤሃ፡ ርሰነ፡ መለኮት፡ ውሉድ፡ ወሉድ፡ እማእላፍ፡ ርእሱ፡ ወርቅ፡ ቂፋዝ፡ </incipit>
+                       <explicit xml:lang="gez"></explicit>
+                       <note>Musical notation above the written lines of the main text.</note>
```

Comment:
here and later, instead of note, this information should be given in `<notatedMusic><desc>`